test/FsAutoComplete.Tests.Lsp/FindReferencesTests.fs

```diff
@@ -458,8 +458,16 @@ let private rangeTests state =
     async {
       let (source, cursors) = sourceWithCursors |> extractRanges
       let! (doc, diags) = server |> Server.createUntitledDocument source
+
+      let pertinentDiags =
+        diags
+        |> Array.filter (function
+          // Ignore extra parens diagnostics here.
+          | { Code = Some "FSAC0004" } -> false
+          | _ -> true)
+
       use doc = doc
-      Expect.hasLength diags 0 "There should be no diags"
+      Expect.hasLength pertinentDiags 0 "There should be no diags"
```

Comment:
It looks like you added this because the source code had unnecessary parens, but you also fixed up the source - maybe revert this change if that's the case?